include/pa_asio.h

```diff
@@ -70,6 +70,26 @@ extern "C"
 PaError PaAsio_GetAvailableBufferSizes( PaDeviceIndex device,
         long *minBufferSizeFrames, long *maxBufferSizeFrames, long *preferredBufferSizeFrames, long *granularity );
 
+/** Register callback to detect changes to buffer size.
+
+ This correponds to the ASIO kAsioBufferSizeChange message.  Note that an ASIO
+ driver may send a more general reset request when the user changes the buffer
+ size.
+
+ @param callback The function to call when ASIO buffer size changes.
```

Comment:
Can this be called with nullptr to Unregister the callback?

Reply:
Yes, now added a note about that to the doc